# HG changeset patch
# User Alexander Dreyer <adreyer@gmx.de>
# Date 1332364717 -3600
# Node ID 43b3931ceed7b57c4c5385442de6ff45bd0f477c
# Parent  617ce1a715329d16e5ebde0bfdde9c4d09072451
FIX explict name lookups (in template base)

diff -r 617ce1a715329d16e5ebde0bfdde9c4d09072451 -r 43b3931ceed7b57c4c5385442de6ff45bd0f477c libpolybori/include/polybori/orderings/CBlockOrderingFacade.h
--- polybori-0.7.1.p6/src/polybori-0.7/polybori/include/CBlockOrderingFacade.h	Mon Mar 19 22:53:46 2012 +0100
+++ polybori-0.7.1.p6/src/polybori-0.7/polybori/include/CBlockOrderingFacade.h	Wed Mar 21 22:18:37 2012 +0100
@@ -91,7 +91,7 @@
     CacheManager<order_lead_tag> cache_mgr(poly.ring());
     typename base_type::descending_property descending;
 
-    return monom(  dd_block_degree_lead(cache_mgr, blockDegCache, 
+    return base_type::monom( dd_block_degree_lead(cache_mgr, blockDegCache, 
                                         poly.navigation(), m_indices.begin(),
                                         set_type(poly.ring()), descending) );
   }
